e2e/pages/shared-lending-platform/NonAssociatedUserProfile.spec.ts

```diff
@@ -31,3 +32,62 @@ test('Complete User Profile -- No Associations -- process', async ({
     await expect(page.locator('#Summary div').first()).toBeVisible();
   });
 });
+
+test('Complete User Profile with Bad Unicode -- No Associations -- process', async ({
+  page,
+}) => {
+  test.slow();
+
+  await test.step('Fillout Complete User Profile (No Associations) with bad unicode and verify values', async () => {
+    const expectedValues = {
+      firstField: controlUnicode.slice(0, 255),
+      lastField: controlUnicode.slice(0, 255),
+      finField: controlUnicode, // Update with correct value after limit in place
```

Comment:
Great! Let's make a ticket for future work and then link it here, here's an example:
https://github.com/cfpb/sbl-frontend/blob/main/src/pages/Filing/ViewUserProfile/index.tsx#L65-L66